x-pack/plugins/painless_lab/public/application/components/main_controls.tsx

```diff
@@ -87,9 +88,14 @@ export function MainControls({
     </EuiContextMenuItem>,
   ];
 
-  const classes = classNames('painlessLab__bottomBar', {
-    'painlessLab__bottomBar-isNavDrawerLocked': isNavDrawerLocked,
-  });
+  // TODO #64541
+  // Can delete all this class stuff
+  let classes = '';
+  if (isNavLegacy) {
+    classes = classNames('painlessLab__bottomBar', {
```

Comment:
I think we still need to apply `painlessLab__bottomBar` in all cases, so that it gains the z-index style that prevents it from being blocked by the nav.

![image](https://user-images.githubusercontent.com/1238659/80425706-0e544900-8899-11ea-953b-f418b31ba0dc.png)

I suggest we extract the `left: $euiNavDrawerWidthCollapsed;` style into another class called `painlessLab__bottomBar-isNavDrawerUnlocked`. We can apply everything like this:

```js
const classes = classNames('painlessLab__bottomBar', {
  'painlessLab__bottomBar-isNavDrawerLocked': isNavLegacy && isNavDrawerLocked,
  'painlessLab__bottomBar-isNavDrawerUnlocked': isNavLegacy && !isNavDrawerLocked,
});
```

Reply:
CJ and I talked and changed the direction of the popover instead of maintaining conditional CSS
![Screen Shot 2020-04-27 at 19 00 27](https://user-images.githubusercontent.com/4188087/80432501-0c4eb200-88ba-11ea-87d9-4340cbeaf4c4.png)